db.json

```diff
@@ -377,6 +377,23 @@
         }
       ],
       "standards": [
+        {
+          "name": "File System",
+          "href": "https://fs.spec.whatwg.org/",
+          "description": "File system infrastructure and API.",
```

Comment:
Usual format for standard descriptions (instead of workstream descriptions) is "The X Standard provides/describes..."